java.lang.IllegalStateException: context 'docker' already exists

I've followed the steps in the tutorial! all went fine until i tried to install liquibase data. 

`C:\Users\bn.OFFICE>liquibase data install --log-level flag`


```
Starting Liquibase at 17:06:57 (version 4.5.0 #52 built at 2021-09-27 16:19+0000)
Liquibase Version: 4.5.0
Liquibase Community 4.5.0 by Datical
[2021-10-27 17:06:57] SEVERE [liquibase.integration] java.lang.IllegalStateException: context 'docker' already exists
liquibase.exception.CommandExecutionException: java.lang.IllegalStateException: context 'docker' already exists
        at liquibase.command.CommandScope.execute(CommandScope.java:153)
        at liquibase.integration.commandline.CommandRunner.call(CommandRunner.java:45)
        at liquibase.integration.commandline.CommandRunner.call(CommandRunner.java:15)
        at picocli.CommandLine.executeUserObject(CommandLine.java:1953)
        at picocli.CommandLine.access$1300(CommandLine.java:145)
        at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2352)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2346)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2311)
        at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2179)
        at picocli.CommandLine.execute(CommandLine.java:2078)
        at liquibase.integration.commandline.LiquibaseCommandLine.lambda$execute$1(LiquibaseCommandLine.java:315)
        at liquibase.Scope.child(Scope.java:186)
        at liquibase.Scope.child(Scope.java:162)
        at liquibase.integration.commandline.LiquibaseCommandLine.execute(LiquibaseCommandLine.java:287)
        at liquibase.integration.commandline.LiquibaseCommandLine.main(LiquibaseCommandLine.java:81)
Caused by: java.lang.IllegalStateException: context 'docker' already exists
        at io.titandata.titan.providers.ProviderFactory.create(ProviderFactory.kt:133)
        at io.titandata.titan.providers.ProviderFactory.create$default(ProviderFactory.kt:131)
        at org.liquibase.ext.data.InstallCommandStep.run(InstallCommandStep.kt:16)
        at liquibase.command.CommandScope.execute(CommandScope.java:147)
        ... 14 more

Unexpected error running Liquibase: java.lang.IllegalStateException: context 'docker' already exists

liquibase.exception.CommandExecutionException: java.lang.IllegalStateException: context 'docker' already exists
        at liquibase.command.CommandScope.execute(CommandScope.java:153)
        at liquibase.integration.commandline.CommandRunner.call(CommandRunner.java:45)
        at liquibase.integration.commandline.CommandRunner.call(CommandRunner.java:15)
        at picocli.CommandLine.executeUserObject(CommandLine.java:1953)
        at picocli.CommandLine.access$1300(CommandLine.java:145)
        at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2352)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2346)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2311)
        at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2179)
        at picocli.CommandLine.execute(CommandLine.java:2078)
        at liquibase.integration.commandline.LiquibaseCommandLine.lambda$execute$1(LiquibaseCommandLine.java:315)
        at liquibase.Scope.child(Scope.java:186)
        at liquibase.Scope.child(Scope.java:162)
        at liquibase.integration.commandline.LiquibaseCommandLine.execute(LiquibaseCommandLine.java:287)
        at liquibase.integration.commandline.LiquibaseCommandLine.main(LiquibaseCommandLine.java:81)
Caused by: java.lang.IllegalStateException: context 'docker' already exists
        at io.titandata.titan.providers.ProviderFactory.create(ProviderFactory.kt:133)
        at io.titandata.titan.providers.ProviderFactory.create$default(ProviderFactory.kt:131)
        at org.liquibase.ext.data.InstallCommandStep.run(InstallCommandStep.kt:16)
        at liquibase.command.CommandScope.execute(CommandScope.java:147)
        ... 14 more
```

--- a/src/main/kotlin/io/titandata/titan/utils/ProgressTracker.kt
+++ b/src/main/kotlin/io/titandata/titan/utils/ProgressTracker.kt
@@ -20,7 +20,11 @@ class ProgressTracker {
     }
 
     private fun getStyle(): ProgressBarStyle {
-        val term = System.getenv("TERM")
+        val term = try {
+             System.getenv("TERM")
+        } finally {
+            ""
+        }
         // TODO get better list of switches
         return when {
             term.contains("xterm") -> ProgressBarStyle.COLORFUL_UNICODE_BLOCK

